terminal_exeuction/terminal_execution.py

```diff
@@ -0,0 +1,84 @@
+"""This script sets up and runs a simple app network for testing."""
```

Comment:
this module should be moved to utils and the new package/folder should be removed